docs/release-notes/release-notes-0.16.0.md

```diff
@@ -141,6 +141,13 @@ of order is also [fixed](https://github.com/lightningnetwork/lnd/pull/7264).
   `updatenodeannouncement` peers cli which did not allow setting/
    unsetting of feature bits also has been fixed. 
 
+* [Enrich the `Channel` message with a peers' alias in order to retrieve it
+  in `ListChannels`.](https://github.com/lightningnetwork/lnd/pull/7006) This
```

Comment:
incorrect link

Reply:
Thanks for catching this @ellemouton, updated.